api_tests/dry/lightning_routes.ts

```diff
@@ -0,0 +1,67 @@
+// These are stateless tests -- they don't require any state of the cnd and they don't change it
+// They are mostly about checking invalid request responses
+import "chai/register-should";
+import "../lib/setup_chai";
+import { oneActorTest } from "../lib/actor_test";
+import { expect } from "chai";
+
+setTimeout(async function() {
+    describe("Lightning routes", () => {
+        oneActorTest(
+            "lightning-routes-post-eth-lnbtc-return-400",
+            async function({ alice }) {
+                return expect(
+                    alice.cnd.postHanEthereumEtherHalightLightningBitcoin()
+                ).to.eventually.be.rejected.then(error => {
```

Comment:
This is actually not the assertion we want but we can't assert what we want until https://github.com/comit-network/comit-js-sdk/issues/122 is implemented.

I am leaning towards making this assertion do less so that we have more reasons to do the above ticket.